scout/parse/case.py

```diff
@@ -9,6 +9,7 @@
 
 from scout.constants import PHENOTYPE_MAP, REV_PHENOTYPE_MAP, REV_SEX_MAP, SEX_MAP
 from scout.exceptions import ConfigError, PedigreeError
+from scout.parse.config_base import ScoutIndividual, ScoutLoadConfig, VcfFiles
```

Comment:
Same ScoutIndividual
```suggestion
from scout.parse.config_base import ScoutLoadConfig, VcfFiles
```